examples/tutorial/jupyter/execution/pandas_on_ray/test/test_notebooks.py

```diff
@@ -11,97 +11,35 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-import nbformat
-from nbconvert.preprocessors import ExecutePreprocessor
-
-test_dataset_path = "taxi.csv"
-ep = ExecutePreprocessor(timeout=600, kernel_name="python3")
-
-download_taxi_dataset = f"""import os
-import urllib.request
-if not os.path.exists("{test_dataset_path}"):
-    url_path = "https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv"
-    urllib.request.urlretrieve(url_path, "{test_dataset_path}")
-    """
-
-
-def _execute_notebook(notebook):
-    """
-    Execute a jupyter notebook.
-
-    Parameters
-    ----------
-    notebook : file-like or str
-        File-like object or path to the notebook to execute.
-    """
-    nb = nbformat.read(notebook, as_version=nbformat.NO_CONVERT)
-    ep.preprocess(nb)
-
+import os
+import sys
 
-def _find_code_cell_idx(nb, identifier):
-    """
-    Find code cell index by provided ``identifier``.
-
-    Parameters
-    ----------
-    nb : dict
-        Dictionary representation of the notebook to look for.
-    identifier : str
-        Unique string which target code cell should contain.
-
-    Returns
-    -------
-    int
-        Code cell index by provided ``identifier``.
-
-    Notes
-    -----
-    Assertion will be raised if ``identifier`` is found in
-    several code cells or isn't found at all.
-    """
-    import_cell_idx = [
-        idx
-        for idx, cell in enumerate(nb["cells"])
-        if cell["cell_type"] == "code" and identifier in cell["source"]
-    ]
-    assert len(import_cell_idx) == 1
-    return import_cell_idx[0]
+import nbformat
 
+UTILS_PARENT_DIR = os.path.abspath(
```

Comment:
Can we just use the following import below to import required stuff?
```python
from modin.examples.tutorial.jupyter.execution.test.utils import ...
```

Reply:
We can make import relative to modin root directory, but in this case, we will need to put `__init__.py` in each nested directory (in `examples`, `examples/tutorial`, `examples/tutorial/jupyter`, `examples/tutorial/jupyter/execution`), otherwise we won't be capable to access these directories as python modules.

Reply:
Ah, wait. Actually we can do it, but wee need to tune inserted path too. Import path relative to Modin root directory was used.